cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -373,6 +377,24 @@ func gcloudProject() []string {
 	return []string{data.Core.Project}
 }
 
+func setupTermDelay() {
+	terminated := make(chan bool, 0)
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGTERM)
+
+	go func() {
+		<-terminated
```

Comment:
dumb question, why have two goroutines here? Why not just have this line read

`<-sigs`?